lib/Doctrine/ORM/UnitOfWork.php

```diff
@@ -1124,6 +1124,20 @@ public function recomputeSingleEntityChangeSet(ClassMetadata $class, $entity)
         foreach ($actualData as $propName => $actualValue) {
             $orgValue = $originalData[$propName] ?? null;
 
+            if (! empty($class->fieldMappings[$propName]['enumType'])) {
```

Comment:
I'm a bit surprised we don't forbid calls to `empty`, and I'd rather you use something more precise. If you had to avoid it, what would you use?

Reply:
To be honest, I copied it from `UnitOfWork.php:1127`. IMO `isset` construct would fit better here (changed)